### DISCUSSION TITLE
[QoL Feature Request] Add collapsable groups for "Core" and "Hub" in the plugin list

### DISCUSSION THREAD

**Discussion by Intecpsp** [runelite/runelite#18487] [Ideas]
I've been thinking about cleaning up a bunch of installed plugins, but keep getting stopped with the only way of finding the ones I can get rid of would be to right-click each one to see if it's possible. Can a way of separating the core plugins from the ones the user installed from the hub be implemented to assist with this?

To add onto this, could additional sort/filter entries be added? Maybe something like: Alphabetical, Last updated date, Install date, etc.

**Comment by coopermor** [comment]
In the meantime you can put in the search box "Plugin Hub" without quotes and it filters to only hub plugins you have installed.

**Reply by Felanbird** [reply]
or just `hub`

**Reply by Intecpsp** [reply]
Is there a list of these filters somewhere? Or is it a workaround searching the metadata?
